docs/python/source/index.md

```diff
@@ -0,0 +1,72 @@
+# rustyms Python bindings
+
+Python bindings to the [rustyms](https://docs.rs/rustyms/) library for proteomics
+and mass spectrometry.
+
+## Quickstart
+
+Python bindings are provided to several core components of the rustyms library, including:
+
+- {py:class}`~rustyms.Element`: Chemical elements
+- {py:class}`~rustyms.MolecularFormula`: Molecular formulas
+- {py:class}`~rustyms.AminoAcid`: Amino acids
+- {py:class}`~rustyms.Modification`: Amino acid modifications, with support for mass shifts,
+  chemical formulas, Unimod, and PSI-MOD labels
+- {py:class}`~rustyms.AmbiguousModification`: {py:class}`~rustyms.Modification` with ambiguous
+  localization.
+- {py:class}`~rustyms.Fragment`: Theoretical fragment ion
+- {py:class}`~rustyms.SequenceElement`: One position in a peptide sequence with amino acid and
+  modifications
+- {py:class}`~rustyms.LinearPeptide`: Peptide sequence, modifications, and charge, with full
+  [ProForma 2.0](https://proforma.readthedocs.io) support
```

Comment:
It does not have full ProForma support, I want to get there eventually but did not get around to it yet. I would send people to the `pro_forma_grammar.md` file in the root for details on what exactly is supported.